packages/boxel/addon/components/boxel/input/selectable-token-amount/index.css

```diff
@@ -0,0 +1,64 @@
+.boxel-input-selectable-token-amount {
+  --boxel-input-selectable-token-amount-height: var(--boxel-sp-xl);
+  --boxel-input-selectable-token-amount-horizontal-padding: var(--boxel-sp-sm);
+  --boxel-input-selectable-token-amount-input-font-size: var(--boxel-font-size);
+
+  position: relative;
+  width: 100%;
+  font-family: var(--boxel-font-family);
+  font-size: var(--boxel-input-selectable-token-amount-input-font-size);
+}
+
+.boxel-input-selectable-token-amount .boxel-input-selectable-token-amount__input {
+  --boxel-input-height: var(--boxel-input-selectable-token-amount-height);
+
+  height: var(--boxel-input-selectable-token-amount-height);
+  padding-left: var(--boxel-input-selectable-token-amount-horizontal-padding);
+  font: inherit;
+  letter-spacing: inherit;
+}
+
+.boxel-input-selectable-token-amount .ember-power-select-status-icon {
+  background: url("/@cardstack/boxel/images/icons/caret-down.svg") no-repeat;
+  width: 11px;
+  height: 9px;
+  display: inline-block;
+}
+
+.boxel-input-selectable-token-amount .ember-basic-dropdown {
+  display: contents;
```

Comment:
I couldn't find an element that matches this rule. Could we remove it?

Reply:
yes!